src/trace/context/Context.ts

```diff
@@ -36,13 +35,13 @@ export default interface Context {
 
   stop(span: Span): boolean;
 
+  async(span: Span): void;
+
+  resync(span: Span): void;
+
```

Comment:
Could you consider to fill the comments of all methods on this interface? I think those would be helpful for newer contributors, as we plan to release 0.1.0.